src/o365/yammer/commands/group/group-user-remove.ts

```diff
@@ -0,0 +1,150 @@
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import YammerCommand from "../../../base/YammerCommand";
+import request from '../../../../request';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: number;
+  userId?: number;
+  confirm?: boolean;
+}
+
+class YammerGroupUserRemoveCommand extends YammerCommand {
+  constructor() {
```

Comment:
I don't think this is necessary since we're not doing anything specific in the constructor other than calling super() which is done automatically.